### PR TITLE
ENH: Move firmware folder to basil (part of pakage)

### PR DESCRIPTION
**Pull request by themperek**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +22,13 @@
 with open('requirements.txt') as f:
     install_requires = f.read().splitlines()
 
+def package_files(directory):
```
Comment (DavidLP) [inline setup.py]: Educated "3rd" best Stackoverflow answer, that acutally is the best answer ;-)

Comment (themperek) [reply setup.py]: ?

**Comment by themperek** [comment]
Package size should not change since firmware was already before part of it just not installed.

File: setup.py (hunk 1 above)
Comment (DavidLP) [reply setup.py]: https://stackoverflow.com/questions/27664504/how-to-add-package-data-recursively-in-python-setup-py/27664856

Comment (laborleben) [reply setup.py]: Isn't MANIFEST.in enough?

Comment (laborleben) [reply setup.py]: and adding '*.v'  after '*.yaml'.

Comment (laborleben) [reply setup.py]: Also consider that the answers are more than 2 years old. Quite a bit for setuptools.

Comment (themperek) [reply setup.py]: >Isn't MANIFEST.in enough?

As I see it will be part of package but will not be installed during installation.

**Pull request merged**